### PR TITLE
Fix shuttle travel time logic and update related unit tests

### PR DESCRIPTION
**Pull request by Carmoucha**
This PR fixes a bug where the shuttle travel time calculation was inaccurate and returned incorrect wait times. 
- Rewrote time handling logic using the current moment properly.
- Fixed currentStop reference issue.
- Ensured fallback and NaN handling for edge cases.
- Test mode was set to false in shuttleSchedule.js
- Updated and fixed unit tests 

<p align="center">
  
</p>

<p align="center">
  
</p>

### CODE REVIEW

File: myApp/app/utils/shuttleUtils.js [hunk 1]
```diff
@@ -129,13 +130,14 @@ export const estimateShuttleFromButton = async (currentStop) => {
...
+  const shuttleRideTime = await TravelFacade.getGoogleTravelTime(
+    currentStop === "SGW" ? sgwStop : loyolaStop,
+    currentStop === "SGW" ? loyolaStop : sgwStop,
+    "driving"
+  );
 
-  if (isNaN(shuttleRideTime)) {
-    console.error("Calculated shuttle ride time is invalid (NaN).");
+  if (shuttleRideTime === null || isNaN(shuttleRideTime)) {
+    console.error("Google API returned invalid shuttle ride time.");
```
Comment (samjamhen) [inline myApp/app/utils/shuttleUtils.js:140]: Nice validation here! I like that it ensures that the shuttle ride time is a valid number before proceeding, which helps prevent any issues with the invalid API responses we were getting. Really nice improvement

**Review by samjamhen — APPROVED** [review]
